workspaces/ballerina/ballerina-visualizer/src/views/BI/Forms/ForkForm/index.tsx

```
@@ -299,56 +334,115 @@ export function ForkForm(props: ForkFormProps) {
+            }}
+            targetLineRange={targetLineRange}
+            fileName={fileName}
+            popupManager={{
+                addPopup: () => { },
+                removeLastPopup: () => { },
+                closePopup: () => { },
+            }}
+            nodeInfo={{ kind: node.codedata.node }}
+        >
+            <FormStyles.Container>
+                {branches.map((branch, index) => {
```

Biome (2.3.14)

[error] This callback passed to map() iterable method should always return a value.

Add missing return statements so that this callback returns a value on all execution paths.

(lint/suspicious/useIterableCallbackReturn)